gen/definitions/crypto_ikev2_profile.yaml

```diff
@@ -52,6 +52,9 @@ attributes:
   - yang_name: keyring/local/name
     tf_name: keyring_local
     example: test
+  - yang_name: ivrf
+    tf_name: ivrf
```

Comment:
In case tf_name is the same as yang_name, we don't need to provide it explicitly.